x-pack/plugins/security_solution/public/overview/containers/overview_cti_links/use_cti_event_counts.ts

```diff
@@ -18,7 +18,7 @@ export const useCtiEventCounts = ({
   from,
   setQuery,
   to,
-}: ThreatIntelLinkPanelProps) => {
+}: Omit<ThreatIntelLinkPanelProps, 'isThreatIntelModuleEnabled'>) => {
```

Comment:
We might be trying too hard here to reuse these types; if this hook and the component that share(d) props have independent use cases, they should each declare their own props.

Reply:
declared a new type for `CtiEnabledModuleProps`, which is the same `Omit<ThreatIntelLinkPanelProps, 'isThreatIntelModuleEnabled'>` it's used in 2 places and a bit more readable IMHO, let me know what you think